cmd/go-filecoin/chain.go

```diff
@@ -121,7 +121,7 @@ var storeStatusCmd = &cmds.Command{
 		Tagline: "Show status of chain sync operation.",
 	},
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
-		syncStatus := GetPorcelainAPI(env).ChainStatus()
+		syncStatus := GetPorcelainAPI(env).SyncerStatus()
```

Comment:
nit: to keep all existing functionality we should add a chain store status endpoint to porcelain, split this into `sync-status` and then add another command for `store-status` to report on the two status values managed by the chain store.